easybuild/easyconfigs/s/SCOTCH/SCOTCH-6.0.9-foss-2019b.eb

```diff
@@ -0,0 +1,19 @@
+name = 'SCOTCH'
+version = '6.0.9'
+
+homepage = 'https://gforge.inria.fr/projects/scotch/'
+description = """Software package and libraries for sequential and parallel graph partitioning,
+static mapping, and sparse matrix block ordering, and sequential mesh and hypergraph partitioning."""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
```

Comment:
@furstj This should've been pulled "down" to `gompi`, since it doesn't need BLAS/LAPACK/FFTW?

Reply:
@boegel Indeed, it should be gompi. Is it still possible to re-open this pr? How should I do it?

Reply:
Just open a follow-up PR to fix the toolchain (it's fine since this is only in `develop`, not included in an EasyBuild release yet)

Reply:
OK, gompi version is in  #9361

Reply:
my bad - thanks for the quick follow up @furstj !